homework6.js

```diff
@@ -0,0 +1,21 @@
+
+var $BuyButton = document.getElementsByClassName("buy");
+var $request = document.getElementsByClassName("furniture");
+$BuyButton.addEventListener("click", handlerButtonBuy);
+var cart = 0;
+function handlerButtonBuy($request) {
+    for(var i = 0; i < $request.length ; i++) {
+        var requestValue = $request[i].innerHTML;
```

Comment:
Это не очень хороший вариант - ориентироваться на внутренности какого-то блока. Кроме прочего, у Вас в html лежать значения с пробелами, нужно от них избавиться. .trim() обрезает лишние пробельные символы в начале и конце строки
```suggestion
        var requestValue = $request[i].innerHTML.trim();
```